packages/dashboard/src/components/map-app.tsx

```diff
@@ -637,7 +639,20 @@ export const MapApp = styled(
               ))
             : null}
           {currentLevel.images.length > 0 && imageUrl && (
-            <ReactThreeFiberImageMaker level={currentLevel} imageUrl={imageUrl} />
+            <ErrorBoundary
+              fallback={<></>}
```

Comment:
fallback should not be empty

Reply:
Fair enough :boom: 

![image](https://github.com/open-rmf/rmf-web/assets/5383623/7a61a9ca-4ba9-4122-8774-1217675b6bc1)

Reply:
1. Styling could be better
2. No need to display the map if there is an error
3. No need for toast anymore, just add to visual clutter

Reply:
> 1. Styling could be better

Any other styling other than the bare minimum could cause a retrieval from the internet, which will break in an airgapped environment. I just used the theme, and the same font we have added statically to prevent that.

Style for this is also very low priority.

> 2. No need to display the map if there is an error

The issue is only from failing to load an image (it could be missing from cache), so there should be no reason, the rest of the map is not rendered (lifts, doors, etc). This will at least still allow the dashboard to be used.

> 3.

Toast removed

> 4.

Made it short and simple. Anything else regarding the error can be read from console errors.

![image](https://github.com/open-rmf/rmf-web/assets/5383623/daf7a439-c5ee-4c42-b45c-c371caedbb40)

Reply:
I think a missing image is a broken map, but it seems you want to ignore the error and continue? I think the image is an important enough part to just fail the whole map but I'm not totally against continuing either, if that is the case, then I guess we can revert back to the empty map + toast.

Reply:
yeah sounds good. I'm mainly coming from a UX perspective. With a misplaced/missing image based on the URL, we will just be missing the floor plan, whereas users can still monitor where the robots are relative to waypoints.

In this case, we won't "break" production, and the fix would just be to ensure the building map server can be brought up again, without disruption to users.